### PR TITLE
[WB-74] Abstract HTTP server logic into util.

### PR DESCRIPTION
**Pull request by austinkelleher**
Related: https://github.com/windbreaker-io/windbreaker-hooks/pull/8

### CODE REVIEW

**Comment by austinkelleher** [comment]
[Abstract HTTP logic into utils](https://trello.com/c/pMIwkSow/74-abstract-http-logic-into-utils)

File: http/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  logger.info('Registering route: ', method, path)
+  router.register({ path, method, middleware, handler })
+}
+
+/**
+* Creates a generic HTTP server
+*
+* @param options {Object}
+*  - options.routes {Object} - A route object containing { middlware, method, path, handler }
```
Comment (charlieduong94) [inline http/index.js]: Minor: this should be an `{Array}` type.

File: http/util/getCommonMiddleware.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const koaBodyParser = require('koa-bodyparser')()
+
+module.exports = function getCommonMiddleware () {
+  return [
+    koaBodyParser
```
Comment (charlieduong94) [inline http/util/getCommonMiddleware.js]: Not every route will need this (ex. health checks). Not a huge deal, though.

Comment (austinkelleher) [reply http/util/getCommonMiddleware.js]: Yeah, but we talked about this in person and determined that this is not a big deal. Right?

Comment (charlieduong94) [reply http/util/getCommonMiddleware.js]: Yep, nbd.

**Pull request merged**